build.gradle

```diff
@@ -12,12 +12,14 @@ repositories {
 }
 
 dependencies {
-    implementation platform("io.opentelemetry:opentelemetry-bom:1.7.0")
+    implementation platform("io.opentelemetry:opentelemetry-bom:1.11.+")
 
     implementation 'io.opentelemetry:opentelemetry-api'
     implementation 'io.opentelemetry:opentelemetry-sdk'
     implementation 'io.opentelemetry:opentelemetry-exporter-otlp-trace'
 
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Required for `@Nullable` annotation on line
https://github.com/dynatrace-oss/junit-jupiter-open-telemetry-extension/blob/main/src/main/java/com/dynatrace/oss/junit/jupiter/tracing/TracingExtension.java#L65